### PR TITLE
First draft for getting-started page

### PR DESCRIPTION
**Pull request by jayk-u**
Here's a first draft of the getting started page with a few screenshot.
I had several issues with formatting so I'll check that later.
My main goal is to get as many feedback as possible to make it nice for the launch (we may come back on it later).

@saro0h Your experience and insight would be super appreciated here 💙

### CODE REVIEW

**Comment by jayk-u** [comment]
known issue: a link points to a page that does not exist yet in this version (but is in another PR)

File: docs/help/getting-started.md [hunk 1]
```diff
@@ -1,27 +1,56 @@
...
+
+With your documentation ready, you could want to make it more customized. Bump.sh offers several options to do so.
+
+You can setup [Custom Domains](help/custom-domains.md), to make your doc available behind your own URL.
+Several branding options are available: from your logo to [Meta Images](help/meta-images.md) (that helps sharing your doc in a fancy way).
+We also support [Documentation Topics](help/doc-topics.md), that helps you adding rich (con)text to your API doc, sharing the informations your community needs.
+
+![](/files/meta-image-2.png)
+
+## And way much more
```
Comment (paulRbr) [inline docs/help/getting-started.md:49]:
```suggestion
## And way more
```
I think it sounds better doesn't it?

File: docs/help/getting-started.md [hunk 2]
```diff
@@ -1,27 +1,56 @@
...
+
+We do not stop here. Feel free to browse this documentation or directly reach us to find out how to make the most of Bump.sh.
+There's a lot more to cover: from Hubs that groups several API documentation to Branching, hosting several version of a same doc, you'll find many simple options that make a huge change.
+
+The team is always here to listen if you have any feature requests or suggestions. We are here to help.
+
+### We hope you'll enjoy using Bump.sh!
```
Comment (paulRbr) [inline docs/help/getting-started.md:56]: Feels strange to use a “Title 3” when there's no texte behind. What about using only bold?

```suggestion
**We hope you'll enjoy using Bump.sh!**
```

**Review by paulRbr — APPROVED** [review]
Cheers, LGTM

File: docs/help/getting-started.md [hunk 3]
```diff
@@ -1,27 +1,56 @@
 # Deploying Bump.sh
```
Comment (ChristopheDujarric) [inline docs/help/getting-started.md:1]:
```suggestion
# Deploying an API documentation on Bump.sh
```

File: docs/help/getting-started.md [hunk 4]
```diff
@@ -1,27 +1,56 @@
...
-Step 1: Create your space/documentation
-Step 2: Upload your file
-    - upload file directly
-    - use the CLI
-    - try an example
-Step 3: Enjoy
+
+### Step 1: Create your space/documentation
+
+After creating your account, you'll be asked to name your first API documentation.
```
Comment (ChristopheDujarric) [inline docs/help/getting-started.md:9]:
```suggestion
After creating your account, you'll be asked [to create and name](https://bump.sh/docs/new?utm_source=bump&utm_medium=content_hub&utm_campaign=getting_started) your first API documentation.
```
How about adding those direct link anywhere possible (for instance also for Hubs and Orgs)

File: docs/help/getting-started.md [hunk 5]
```diff
@@ -1,27 +1,56 @@
...
+You can also use our examples to see Bump.sh in action. Finally, our [CLI](help/bump-cli) can also be directly used to jump in.
+
+![](/files/upload-spec-file.png)
+
+### Step 3: Enjoy
+
+Generating your first documentation will take a few seconds. But let's not stop here: many features are awaiting to customize and make the most of your API documentation. Let's have a look in the next section.
+
 
 ## Manage Access
```
Comment (ChristopheDujarric) [inline docs/help/getting-started.md:26]:
```suggestion
## Publishing your documentation to the right audience
```
I would try to approach titles more from the user problem than from the feature perspective.

File: docs/help/getting-started.md [hunk 6]
```diff
@@ -1,27 +1,56 @@
...
-- Make it public or private
-- Create an organization
-- Invite teammates
+
+Congratulations on releasing your first API documentation on Bump.sh!
+
+One of the first things you may have in mind is how to share your documentation with the world, your team or keep it for yourself.
+Bump.sh offers several options to manage access to your documentation.
+During the onboarding, you may have noticed the option of making your doc public or private.
```
Comment (ChristopheDujarric) [inline docs/help/getting-started.md:32]: We should probably try to explicit the three types of use cases: public, private and partner.

File: docs/help/getting-started.md [hunk 7]
```diff
@@ -1,27 +1,56 @@
...
+
+Public documentation are available to anyone with the URL or if the search engine indexation has been activated.
+Privates are only accessible to you (or your team/organization if you have one).
```
Comment (ChristopheDujarric) [inline docs/help/getting-started.md:35]:
```suggestion
Private ones are only accessible to you (or your team/organization if you have one).
```
"privates" relate to... something else ;)

File: docs/help/getting-started.md [hunk 8]
```diff
@@ -1,27 +1,56 @@
...
+
+We share more details on the [Access Management page](help/access-management.md).
 
 ## Customize it
```
Comment (ChristopheDujarric) [inline docs/help/getting-started.md:39]:
```suggestion
## Add your brand and colors
```

**Comment by saro0h** [comment]
I have a general remark on the fact that we never explain to the user that everything is based on a specification (generated API schema). That is not explicit that the documentation must be written before beginning with bump.
Yesterday I discussed with a developper that discovered completely the product, and he kept saying "but how do you spot that I changed the code of my API and it it needs more or less documentation?". He really didn't see the how the diff could be displayed.

I don't know if this insight should be taken into account for this iteration.

To sum up:
  1. The how it actually work to spot the "changes" between two deploys is not clear.
  2. As a new comer to the documentation , I don't see right away how it looks and where I can see the diff in the app without actually figure it out myself.

File: docs/help/getting-started.md (hunk 5 above)
Comment (jayk-u) [reply docs/help/getting-started.md:26]: I'll suggest something a bit different ("right audience", are you trying to hurt me?)
